formatter/json.go

```diff
@@ -77,8 +77,8 @@ func (f *Formatter) jsonPrint(issues tflint.Issues, tferr *tflint.Error) {
 
 	if tferr != nil {
 		var errs []error
-		var diags *hcl.Diagnostics
-		if errors.As(tferr.Cause, &diags) {
+		var diags hcl.Diagnostics
+		if errors.As(tferr.Cause, diags) {
```

Comment:
Per the error here, this still should be a pointer, `&diags`. Before, you had a pointer to a pointer. Now, you'll have a pointer to a value.